### DISCUSSION TITLE
Can someone share precompiled libge0 for linux x64?

### DISCUSSION THREAD

**Discussion by EvgenyZaretskiy** [organicmaps/organicmaps#9220] [Q&A]
I want to use it to encode coordinates to URL short link format for Organic Maps. I tried to compile all from sources following instructions provided here https://github.com/organicmaps/organicmaps/blob/master/docs/INSTALL.md, but unsuccessfully. 

Used https://github.com/organicmaps/organicmaps/blob/master/ge0/url_generator.cpp and some part of my code, but it requires libge0.  

Really I want to make a tiny app which takes latitude, longitude and produce short link URL in Organic Maps format.

**Comment by RicoElectrico** [comment]
> I tried to compile all from sources following instructions provided here https://github.com/organicmaps/organicmaps/blob/master/docs/INSTALL.md, but unsuccessfully.

Can you paste the error message and your setup details? Maybe some step in the docs is not well documented or outdated and updating the instructions would benefit anyone.
